data/notices.json

```
@@ -1,5 +1,17 @@
 {
   "notices": [
+    {
+      "title": "aws-iam: IAM OIDC provider used with EKS retrieves expiring certificate",
+      "issueNumber": 8607,
+      "overview": "An issue in the `aws-iam` module's `OpenIdConnectProvider` may break your application by the end of the year. Please see the issue thread for a workaround if you cannot upgrade when the fix is released.",
+      "components": [
+        {
+          "name": "framework",
+          "version": "<=2.50.0"
```

We didn't make the 2.50.0 release.

```suggestion
          "version": "<=2.51.0"
```

Also add v1 notice with version 1.181.0